mavros/src/lib/uas_data.cpp

```diff
@@ -51,9 +51,14 @@ UAS::UAS() :
 		ros::shutdown();
 	}
 
-	// send static transform from "local_origin" (ENU) to "local_origin_ned" (NED)
-	publish_static_transform("local_origin", "local_origin_ned", Eigen::Affine3d(ftf::quaternion_from_rpy(M_PI, 0, M_PI_2)));
-	publish_static_transform("fcu", "fcu_frd", Eigen::Affine3d(ftf::quaternion_from_rpy(M_PI, 0, 0)));
+	// send static transform that are part of the tf tree as a vector
+	std::vector<geometry_msgs::TransformStamped> transform_vector;
+	add_static_transform("map", "map_ned", Eigen::Affine3d(ftf::quaternion_from_rpy(M_PI, 0, M_PI_2)),transform_vector);
+	add_static_transform("odom", "odom_ned", Eigen::Affine3d(ftf::quaternion_from_rpy(M_PI, 0, M_PI_2)),transform_vector);
+	add_static_transform("base_link", "base_link_frd", Eigen::Affine3d(ftf::quaternion_from_rpy(M_PI, 0, 0)),transform_vector);
+
+	tf2_static_broadcaster.sendTransform(transform_vector);
+	ROS_WARN_STREAM("Publish static");
```

Comment:
@kamilritz the ROS_WARN_STREAM is still appearing

Reply:
thanks, it is removed now